tst/EnergyPlus/unit/AirTerminalSingleDuctMixer.unit.cc

```diff
@@ -7693,7 +7694,7 @@ TEST_F(EnergyPlusFixture, AirTerminalSingleDuctMixer_SimFCU_ATMInletSideTest)
 
     state->dataSize->ZoneEqSizing.allocate(1);
     auto &zoneEqSizing(state->dataSize->ZoneEqSizing(1));
-    zoneEqSizing.SizingMethod.allocate(DataHVACGlobals::NumOfSizingTypes);
+    zoneEqSizing.SizingMethod.allocate(static_cast<int>(AutoSizingType::Num));
```

Comment:
The unit tests have lots and lots of allocations of this Array1D.  Once I make it an array these will just vanish.